pkg/framework/context/context.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package context
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/didcomm/protocol/exchange"
```

Comment:
ditto: https://github.com/hyperledger/aries-framework-go/pull/88/files#r313588529

Reply:
@troyronda [changed the abstraction](https://github.com/hyperledger/aries-framework-go/pull/88#discussion_r313604198)